### DISCUSSION TITLE
Local connection is UDP first, then TCP?

### DISCUSSION THREAD

**Discussion by ColtonIdle** [parnic/node-screenlogic#75] [General]
Just wanting to make sure that I understand how this library works. I'm only interested in a local connection which is done over udp . But it seems like once the service is found via udp 255.255.255.255 1444 then you get an ip + port... which you can then use tcp to connect to?

Or do i have something backwards here? Thanks in advance!

**Comment by parnic** [comment]
Yes that's how it works. The device can only be found via udp broadcast, specifically.

**Comment by ColtonIdle** [comment]
Nice. I'm actually way more familiar with tcp so that's good to know. At first I thought it was all udp. Cheers

Sent from Proton Mail Android

-------- Original Message --------
